tests/installation/accept_license.pm

```diff
@@ -36,7 +36,7 @@ sub run {
     # optional checks for the extended installation
     if (get_var('INSTALLER_EXTENDED_TEST')) {
         $self->verify_license_has_to_be_accepted;
-        $self->verify_license_translations unless is_sle('15+');
+        $self->verify_license_translations
```

Comment:
Not sure how VR worked, but you miss `;` =)

Reply:
Also, be aware that this scenario is enabled for TW: https://openqa.opensuse.org/tests/latest?arch=x86_64&distri=opensuse&flavor=DVD&machine=64bit&test=installer_extended&version=Tumbleweed

Your change will break it.

Reply:
But it doesnt use accept_license in tw